main.go

```diff
@@ -29,12 +29,20 @@ import (
 
 	clientset "github.com/xmudrii/etcdproxy-controller/pkg/client/clientset/versioned"
 	informers "github.com/xmudrii/etcdproxy-controller/pkg/client/informers/externalversions"
+	"github.com/xmudrii/etcdproxy-controller/pkg/controller/corecerts"
+	"github.com/xmudrii/etcdproxy-controller/pkg/controller/etcdproxy"
 	"github.com/xmudrii/etcdproxy-controller/pkg/signals"
 )
 
 var (
 	masterURL  string
```

Comment:
I don't think you should have this option.  I think you should just accept a --kubeconfig and then use a method like https://github.com/openshift/library-go/blob/master/pkg/config/client/client_config.go#L15 to either use the file or the in-cluster-config.

You may want to just copy that instead of trying to pull in the entire openshift api surface.